keyboards/idobao/id61/specialk.c

```diff
@@ -0,0 +1,77 @@
+// Copyright 2022 Vino Rodrigues (@vinorodrigues)
```

Comment:
This appears to only be used in the idobao keymap.  These files should be moved into that folder.